### ISSUE TITLE
Failed to stop tracking time

### ISSUE THREAD

**Issue by piddyt-fambyt** [dominikbraun/timetrace#85]
Hi there,

thanks for the project. It looks nice.
However I encountered a bug trying to stop tracking:

```
timetrace create project test
✔️ Created project test
timetrace start test
✔️ Started tracking time
timetrace stop
❗ Failed to stop tracking: record already exists
```
timetrace status works fine.

The json looks like this:
```json
{
	"start": "2021-05-26T10:23:37.5685406+02:00",
	"end": null,
	"project": {
		"key": "test"
	},
	"is_billable": false
}
```

I use timetrace version 0.6.0 on Windows 10.

**Comment by dominikbraun** [comment]
Can confirm this bug with timetrace v0.6.0 on macOS 11.3.

**Comment by dominikbraun** [comment]
This bug has been fixed with [release 0.6.1](https://github.com/dominikbraun/timetrace/releases/tag/v0.6.1). Just download the latest release,`timetrace stop` should work again.
